### ISSUE TITLE
[error] TypeError: The "this" argument must be an instance of Performance. Received [Object]

### ISSUE THREAD

**Issue by TheSolly** [GoogleCloudPlatform/cloud-code-vscode#906]
**Environment**:
*(Hint: "Report Extension Issue on Github" command will fill these out for you.)*

**Version information**

Cloud Code Extension version: 2.10.0

VSCode version:  1.90.0-insider (Universal)

OS:  MacOS

Cloud SDK:

Skaffold: 

Kubectl:

**Description**:

Whenever I install the extension and try to log in, I get this error, I tried to change my system node version to v16, but didn't solve the issue.

<img width="1410" alt="image" src="https://github.com/GoogleCloudPlatform/cloud-code-vscode/assets/31882844/1ec7280c-b353-4b9b-bca6-4fa0960027a4">

**Comment by dpanigo** [comment]
I have exaclty the same issue: The "this" argument must be an instance of Performance. Received [Object]; and I'm also using VS Code Insiders.
The Gemini extension loggin button does nothing.
Any other method to loggin into google cloud fails.
I've reinstalled the extension. I've also tried to install previous releases... Nothing works
Notwhithstanding, my error log (at Toggle Developper Tools -> Console) is slightly different:


Error: The "this" argument must be an instance of Performance. Received [Object]
    at o.h (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:153:192010)
    at o.$executeContributedCommand (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:153:192523)
    at s.S (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:5505)
    at s.Q (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:5271)
    at s.M (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:4361)
    at s.L (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:3440)
    at n.value (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:2227)
    at f.y (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:82:664)
    at fire (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:82:881)
    at c.fire (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:107:14122)
    at n.value (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:176:8020)
    at f.y (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:82:664)
    at fire (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:82:881)
    at c.fire (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:107:14122)
    at MessagePortMain.<anonymous> (c:\Users\user\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:176:6300)
    at MessagePortMain.emit (node:events:514:28)
    at MessagePortMain.emit (node:domain:488:12)
    at Object.MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:2285)

**Comment by nicsuzor** [comment]
Confirmed also on Windows running vscode insiders in a fresh devcontainer using docker WSL2.
```
TypeError: The "this" argument must be an instance of Performance. Received [Object]
    at t.Metrics.now (node:internal/perf/performance:139:5)
    at t.Metrics.withMetrics (/home/nic/.vscode-server-insiders/extensions/googlecloudtools.cloudcode-2.10.0-insiders.0/dist/extension.js:14:10879911)
```

**Comment by AF250329** [comment]
I even asked it here:  https://stackoverflow.com/questions/78486276/vscode-gemini-code-assistant-failed-to-load

**Comment by AF250329** [comment]
new version updated today: Version 2.11.1 (May 2024)   and this bug this exist

![image](https://github.com/GoogleCloudPlatform/cloud-code-vscode/assets/78792938/8b228cbe-39c0-4e3a-b1d7-57ee7be143ea)

**Comment by galz10** [comment]
Hi all, thanks for reporting this bug, this issue should now be fixed on 2.11.3.
